File: Python/tarjeta.py
from payment import Payment


class Tarjeta(Payment):
    number = str
    cvv = str
    date = str

    def __init__(self, number, cvv, date) -> None:
        super().__init__()
        self.number = number
        self.cvv = cvv
        self.date = date

File: PHP/tarjeta.php
<?php
require_once('payment.php');
class Tarjeta extends Payment
{
    public $number;
    public $cvv;
    public $date;

    public function __construct($number, $cvv, $date)
    {
        $this->number = $number;
        $this->cvv = $cvv;
        $this->date = $date;
    }
}
